pcs/resource.py

```diff
@@ -813,6 +851,7 @@ def resource_ban(lib, argv, modifiers):
         raise CmdLineInputError()
     resource_id, node, lifetime = _parse_resource_move_ban(argv)
 
+    _check_is_not_stonith(lib, [resource_id])
```

Comment:
Can this be moved to a library command? Then we could simply replace deprecation warning with an error in order to actually forbid banning stonith resources.